commands/createfixpullrequests.go

```diff
@@ -330,17 +339,21 @@ func (cfp *CreateFixPullRequestsCmd) addVulnerabilityToFixVersionsMap(vulnerabil
 			return err
 		}
 		// First appearance of a version that fixes the current impacted package
-		fixVersionsMap[vulnerability.ImpactedDependencyName] = utils.NewFixVersionInfo(vulnFixVersion, vulnerability.Technology, isDirectDependency)
+		fixVersionsMap[vulnerability.ImpactedDependencyName] = utils.NewFixDetails(vulnFixVersion, vulnerability.Technology, isDirectDependency, vulnerability.ImpactedDependencyName)
 	}
 	return nil
 }
 
-func (cfp *CreateFixPullRequestsCmd) updatePackageToFixedVersion(impactedPackage string, fixVersionInfo *utils.FixVersionInfo) (err error) {
+// Updates impacted package
+// ShouldFix will return false when we don't want to update the package, for example,
+// 1. Not supported Indirect dependency fix
```

Comment:
Not supported Indirect dependency fix
-->
Unsupported fixes for Indirect dependencies for the package manager